site/write_ups/2023/25-01-23_writeup.md

```diff
@@ -0,0 +1,108 @@
+    
+Natalie, ambitious as ever, wants to write up the Data Hazards paper (with some excellent co-authors!), do some more data ethics teaching (inspired by Huw's outreach last year and Euan's lecturing this year). On Natalie's reading list is "Kang, E. B. (2023), Ground truth tracings (GTT): On the epistemic limits of machine learning and [Big Data & Society, 10(1)](https://doi.org/10.1177/20539517221146122") - recommended by Ola. In new projects, Natlie wants to do research that is a positive thing even if the result aren't interesting, set up a dedicated area for the blog on the website, try a weird (not first past the post) voting system for data ethics club and would love to write another paper with the data ethics club crowd!
+
+Zoë wants to interview the Data Ethics organisers on the NHS-R Community podcast that she started being involved with last year. She is planning to introduce/mention Data Hazards in workshops/meetings to raise awareness and try to tie it into everyday work that analysts in the NHS/Councils do. She also is going to research/read more about [Palantir](https://www.bbc.co.uk/news/business-54348456 and https://www.theguardian.com/society/2022/jun/21/palantir-concerns-over-data-firm-poised-to-be-operating-system-of-nhs) and the possible implications for the work at the National level with their data platforms, particularly linked to the recent announcement of working with Posit (formerly RStudio). This might be something that could be discussed in an ethics committee type format.
+
+Alessia plans to pilot similar work at Wellcome Genome Campus to facilitate data scientists (and other professionals) to engage with ethical issues relevant to their work! 
+
+Euan wants to find ways to introduce more Data Ethics material (and memes, of course) in lectures/tutorials for students and staff at his work. Euan also wants to successfully navigate academia collaboratively, rejecting the competition paradigm (slightly more ambitious than remembering to say "please" and "thank you" to your at home Google assistant!)
+
+Our very own Nina wants to find a way to spread the word about data ethics to other fields using data (e.g. psychology! medicine!) - this might be by running a data hazards workshop. She would also like to find a way to make this formally part of my job, rather than just a passion project. Nina would also like to be brave enough to engage more with the data ethics community, maybe by attending a conference?
```

Comment:
```suggestion
Our very own Nina wants to find a way to spread the word about data ethics to other fields using data (e.g. psychology! medicine!) - this might be by running a data hazards workshop. She would also like to find a way to make this formally part of her job, rather than just a passion project. Nina would also like to be brave enough to engage more with the data ethics community, maybe by attending a conference?
```